.github/workflows/test-and-demo.yaml

```diff
@@ -35,9 +35,9 @@ jobs:
           behavioral:
             - 'vue-components/**'
     - name: Set up node
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2
```

Comment:
Is it intentional that most of the other setup-node lines are still at v1 AFAICT?